src/DIRAC/WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -43,6 +43,7 @@
 from DIRAC.DataManagementSystem.Client.DataManager import DataManager
 from DIRAC.DataManagementSystem.Client.FailoverTransfer import FailoverTransfer
 from DIRAC.DataManagementSystem.Utilities.DMSHelpers import resolveSEGroup
```

Comment:
Then this is not used, at least in this module. It is still used in other places, though, should those be consider for an update?

Reply:
correct

Reply:
For the other places, not necessarily, it only makes sense for job upload